tool/tctl/common/resource_command.go

```diff
@@ -908,7 +971,7 @@ func (rc *ResourceCommand) createOktaImportRule(ctx context.Context, client auth
 			return trace.Wrap(err)
 		}
 	}
-	fmt.Printf("Okta import rule '%s' has been %s\n", importRule.GetName(), UpsertVerb(exists, rc.IsForced()))
+	fmt.Printf("okta import rule '%s' has been %s\n", importRule.GetName(), UpsertVerb(exists, rc.force))
```

Comment:
```suggestion
	fmt.Printf("Okta import rule '%s' has been %s\n", importRule.GetName(), UpsertVerb(exists, rc.force))
```

Okta is the name of a company, it should be capitalized in user-facing messages